src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -472,19 +472,16 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None):
                             data['nskyideal'][i], data['status'][i],
                             data['betaper'][i]) = aphot
 
+                # Delete unsaveable FITS data
+                del (data.attrs['filename'], data.attrs['mhdr'],
```

Comment:
I think we solved this problem, so there should no longer be any need to delete these attributes woth that latest version.

Reply:
The code before this commit worked fine unless you don't save the FluxData and then try to save the SpecData. I need to do that because the GO data I'm working on ended up consuming ~1 TB with FluxData save files while I was doing a parameter exploration...

Reply:
Is there a better solution that doesn't always delete these parameters? I'm on my phone, so I can't look at it easily.

Reply:
Sure, I had just figured those FITS header details weren't worth saving, but I'll just make another quick commit to try to save them when possible and delete them if not

Reply:
Alright, done